### PR TITLE
Add rails dbconsole

### LINKED ISSUE #10843: Add rails dbconsole command to the documentation
**Is your feature request related to a problem? Please describe.**

This is a documentation update suggestion from @citizen428.

The suggestion is to add to the documentation another way to get to the database prompt (currently psql) by running `rails dbconsole`.

**Additional context**

For more context, see https://github.com/forem/forem/pull/10835/files#r504378131

### PR DESCRIPTION
**Pull request by ShubhankarKG**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

Adds `rails dbconsole` to open up database terminal, and explains why it is a better alternative.

## Related Tickets & Documents

Fixes #10843 

## QA Instructions, Screenshots, Recordings

Screenshot from 2020-10-15 13-43-37

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [x] docs.forem.com
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

File: docs/faqs.md [hunk 1]
```diff
@@ -34,8 +34,13 @@ comments locally, you will need to update the score of your local comments
 manually. Here's how:
 
 1. Open the terminal.
-2. Run `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL
+2. Run `rails dbconsole` to open the PostgreSQL terminal. Alternatively, run
+   `psql PracticalDeveloper_development` to open `psql`, the PostgreSQL
    terminal.
+
+> Note: dbconsole reads database information from config/database.yml which is
+> always better since database configs might change in the future.
```
Comment (jacobherrington) [inline docs/faqs.md:42]: I like the spirit of explaining this, but can we move this down beneath the list of steps? I think the break from the steps is a bit confusing.

What do you think?

**Review by jacobherrington — COMMENTED** [review]
Thanks for improving the docs! I like this idea, I had a quick question that I left as a comment. :smile:

File: docs/faqs.md (hunk 1 above)
Comment (ShubhankarKG) [reply docs/faqs.md:42]: Totally agreed with breaking the flow from an user perspective. The note stands there to explain why we want users to go to the first alternative, but it didn't strike me that as an user, I can read the note later as well. I'll update this in the next commit.

**Review by jacobherrington — APPROVED** [review]
I think this is an improvement! Thanks for making this PR @ShubhankarKG, we always appreciate improvements to the documentation. :scroll: :+1:

**Review by benhalpern — APPROVED** [review]

**Pull request merged**